parallel-cluster-post-install.sh

```diff
@@ -36,29 +44,41 @@ else
 fi
 
 # Get the cryoSPARC license ID, optional path, and optional versions from the script arguments
-CRYOSPARC_LICENSE_ID=$2
-CRYOSPARC_INSTALL_PATH=${3:-/shared/cryosparc}
-CUDA_INSTALL_PATH=${4:-/shared/cuda}
-CUDA_VERSION=${5:-11.3.1}
-CUDA_LONG_VERSION=${6:-11.3.1_465.19.01}
-CRYOSPARC_VERSION=${7:-latest}
+CRYOSPARC_LICENSE_ID=$1
+CRYOSPARC_INSTALL_PATH=${2:-/shared/cryosparc}
+CUDA_INSTALL_PATH=${3:-/shared/cuda}
+CUDA_VERSION=${4:-11.3.1}
+CUDA_LONG_VERSION=${5:-11.3.1_465.19.01}
+PROJECT_DATA_PATH=${6:-/fsx}
+
+/bin/su -c "mkdir -p ${PROJECT_DATA_PATH}/validate-lanes" - ${OSUSER} || chmod 777 ${PROJECT_DATA_PATH}
 
 # Install the AWS CLI
 pip3 install --upgrade awscli boto3
 
-yum -y update
+set -e
+
+#yum -y update
```

Comment:
Did you mean to leave this commented out? Is it optional?

Reply:
Who commented it out then?  :)  We can either uncomment it and re-verify (erring on the side of the most recent dependencies) or remove it and force users to either use the versions we already have or run it themselves manually. If the result is the same I'd go for the former as it's my preference.

Reply:
The latter is a better option in my opinion, since the installation process already takes so long. I deleted it in the latest commit.